src/main/java/com/openClassroomsProject/SafetyNetAlerts/service/impl/PersonServiceImpl.java

```diff
@@ -114,4 +116,38 @@ public boolean deleteAPerson(UniqueIdentifier uniqueIdentifier) {
         personInformation.setAllergies(medicalRecord.get().getAllergies());
         return Optional.of(personInformation);
     }
+
+    @Override
+    public Optional<ChildrenAndOtherMembers> getListOfChildrenLivingAtThisAddress(String address) {
```

Comment:
This fun is too long, could have been great to split it in sub private functions :)

Reply:
Like that ?